packages/asset-buyer/src/types.ts

```diff
@@ -121,3 +126,11 @@ export interface OrdersAndFillableAmounts {
     orders: SignedOrder[];
     remainingFillableMakerAssetAmounts: BigNumber[];
 }
+
+/**
+ * Represents available liquidity for a given assetData
+ */
+export interface LiquidityForAssetData {
+    tokensAvailableInBaseUnits: number;
+    ethValueAvailableInWei: number;
```

Comment:
`amountAvailableInWei`, `valueAvailableInWei` ?

Reply:
Is "wei" an appropriate term for an erc20 token which could have a decimal amount that is different than 18?  I prefer the current naming, but could be swayed if you feel strongly or if there is an established pattern here